### PR TITLE
BS5 work

### LINKED ISSUE #47: Fix dropdown colors on sm-md devices
When I navigate to https://docs.ropensci.org/rtweet/ on my smartphone, the color of the navbar dropdown links make difficult to see the text:

image

### PR DESCRIPTION
**Pull request by maelle**
Fix #47

TODOS

- [x] waiting on pkgdown reg head customization
- [x] #53

Summary
+ moves the CSS to extra.scss
+ uses a include template for the header stuff we need for Matomo etc
+ uses an include to add the logo to the navbar without having to maintain a navbar template

### CODE REVIEW

File: inst/pkgdown/templates/navbar-package-version.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<a href="https://ropensci.org"><img src="https://ropensci.org/img/icon_short_white.svg" id="hexlogo" alt="rOpenSci"></a>
```
Comment (maelle) [inline inst/pkgdown/templates/navbar-package-version.html]: the exact way we'll add this customization depends on https://github.com/r-lib/pkgdown/pull/1887

Comment (maelle) [reply inst/pkgdown/templates/navbar-package-version.html]: to be updated depending on https://github.com/r-lib/pkgdown/pull/1891

**Comment by maelle** [comment]
https://github.com/r-lib/pkgdown/pull/1923

**Comment by jeroen** [comment]
Small initial observations:

 - I see whitespace underneath the footer that probably should not be there.
 - Also I get horizontal scroll (on chrome)
Screen Shot 2021-11-30 at 12 01 18 PM


For articles:

 - In width: sm the margin for code blocks is missing
Screen Shot 2021-11-30 at 11 58 11 AM

**Comment by jeroen** [comment]
With `targets` the menu bar seems to have the wrong colors. 

Screen Shot 2021-11-30 at 1 22 46 PM

**Comment by maelle** [comment]
reg targets the problem is the specification of type default of navbar in the pkgdown config file. is it a customization we should override @jeroen (i.e. not let users choose the navbar type)

**Comment by dieghernan** [comment]
Sorry for jump in! It would be really cool (IMHO) if you can use the same font than in https://ropensci.org/.

Looking at the source code on https://github.com/ropensci/roweb3/tree/master/themes/ropensci/static/fonts/proxima it seems that is Proxima Nova.

The question to me is if this is possible, as that font seems to be a paid font: https://www.marksimonson.com/fonts/view/proxima-nova

**Comment by maelle** [comment]
@dieghernan Thank you for the suggestion! We are discussing this internally.

**Pull request merged**